config.md

```diff
@@ -25,6 +25,8 @@ For example, if an implementation is compliant with version 1.0.1 of the spec, i
 **`root`** (object, required) configures the container's root filesystem.
 
 * **`path`** (string, required) Specifies the path to the root filesystem for the container.
+  The path MUST be an a relative path (not starting with /), which is relative to the bundle. e.g.
```

Comment:
We used to have a relative `path` restriction, but it was (wisely, I think) lifted in #394.  There is in-flight work in #469 to finish off the work that #394 started, but I don't think we want to go back to requiring relative paths here.